build-server.sh

```diff
@@ -34,12 +37,8 @@ while [[ $# -gt 0 ]]; do
 done
 
 docker pull ghcr.io/aica-technology/ros2-control-libraries:"${ROS_VERSION}"
-DOCKER_BUILDKIT=1 docker build --target development -t "${IMAGE_NAME}/development:latest" "${BUILD_FLAGS[@]}" .
-
-if [ "${BUILD_PROD}" = true ]; then
-  DOCKER_BUILDKIT=1 docker build --target production -t "${IMAGE_NAME}:latest" "${BUILD_FLAGS[@]}" .
-fi
+DOCKER_BUILDKIT=1 docker build -t "${IMAGE_NAME}/${IMAGE_TAG}" "${BUILD_FLAGS[@]}" . || exit 1
```

Comment:
Name and tag should be colon separated, same applies to line 43 for serving
```suggestion
DOCKER_BUILDKIT=1 docker build -t "${IMAGE_NAME}:${IMAGE_TAG}" "${BUILD_FLAGS[@]}" . || exit 1
```